### DISCUSSION TITLE
Font size too small in script interface

### DISCUSSION THREAD

**Discussion by amisano** [dbeaver/dbeaver#16193] [General]
Hi to all
I need a solution to this problem 
I have a very strange situation on font size on sql script input, because the characters on the screen have reduced in size and become unreadable.
I probably used some buttons or settings by mistake.
I couldn't find any option to set the font size.
Can you help me ?
thank you
Version 22.0.2
eamisano@guala.net

**Comment by ShadelessFox** [comment]
Hello @amisano,

Are you talking about the font size in the SQL Editor?

You can adjust its size using <kbd>Ctrl+-</kbd> and <kbd>Ctrl+=</kbd> keyboard shortcuts.

**Reply by Jonston** [reply]
How to increase the interface font?CTRL+- not work.i have windows 11

**Reply by ShadelessFox** [reply]
The provided shortcut only affects text editors.

To increase font size of the interface, go to <kbd>Window</kbd> &rArr; <kbd>Preferences</kbd> &rArr; <kbd>User Interface</kbd> &rArr; <kbd>Appearance</kbd> &rArr; <kbd>Labels and Fonts</kbd> and tweak font sizes for individual components.

**Reply by Quincy123p** [reply]
Has the editor font size issue been resolved with v22.2.5? Do not want to change windows font settings.  Just want to make editor font size larger when needed in UI.

**Reply by ShadelessFox** [reply]
What issue are you talking about? There were no issues regarding font size in the SQL editor.

**Reply by Kyogre** [reply]
Maybe that will help somebody. On Windows you can change font of the SQL editor via this key combos:
1. Ctrl + "+" on the number line (not numpad!)
2. Ctrl + Shift + "+" on numpad
Same goes for minus button.

**Reply by gmcintyre-conga** [reply]
cannot change the font size in SQL Console or sql editors. None of these suggestions work. There is no "Labels & Fonts" under appearance, CTRL + +, CTRL+SHIFT++, nothing. I'm using a Mac and trying CMD instead of CTRL, but no dice.

**Reply by E1izabeth** [reply]
There is a mistake in @ShadelessFox suggestion. The preference page is called Colors and Fonts. You can write a word "font" to the search text area to quickly find it.

**Reply by jeff-bowles** [reply]
@E1izabeth and @ShadelessFox ,
Is it intentional that the SQL entry window does not have it's own "Colors and Fonts" setting?  I see there is a unique way to change the font size, which makes me think the omission is by design.  If so, is there a specific reason?  I would like to use a ligature font when writing SQL.

**Reply by E1izabeth** [reply]
I'm not sure I fully understand your question, but I could say that to change the font for SQL Editor, you need to go to `Preferences->User Interface->Appearance->Colors and Fonts`, find `DBeaver Fonts` folder and change `Monospace font` there.

![image](https://github.com/dbeaver/dbeaver/assets/28875055/15786382-4baf-4911-aaab-1bff33c9ca37)

**Reply by Linusyao** [reply]
> There is a mistake in @ShadelessFox suggestion. The preference page is called Colors and Fonts. You can write a word "font" to the search text area to quickly find it.

Es funktioniert! Vielen Dank!

**Comment by amisano** [comment]
Ciao
Yes I have resolved
Thanks for your support !


Da: Quincy123p ***@***.***>
Inviato: domenica 11 dicembre 2022 16:03
A: dbeaver/dbeaver ***@***.***>
Cc: Amisano Enrico ***@***.***>; Mention ***@***.***>
Oggetto: Re: [dbeaver/dbeaver] Font size too small in script interface (Discussion #16193)


Has the editor font size issue been resolved with v22.2.5? Do not want to change windows font settings. Just want to make editor font size larger when needed in UI.

**Comment by amisano** [comment]
Was An old issue
A question i Have post to increase font on sql editor
Was solved with control + As in a browser sorry




Inviato dal mio Galaxy



-------- Messaggio originale --------
Da: ShadelessFox ***@***.***>
Data: 11/12/22 17:16 (GMT+01:00)
A: dbeaver/dbeaver ***@***.***>
Cc: Amisano Enrico ***@***.***>, Mention ***@***.***>
Oggetto: Re: [dbeaver/dbeaver] Font size too small in script interface (Discussion #16193)


What issue are you talking about? There were no issues regarding font size in the SQL editor.

**Comment by Quincy123p** [comment]
Yes SQL editor allows font sizing, It’s the Db window that cannot be sized with Ctrl-+ or zoomed in Dbeaver. Do not want to change the windows settings on the desktop, that effects all apps. How can Dbeaver Db window font size or zoom be enabled on the Db window in Dbeaver?

**Reply by E1izabeth** [reply]
You can increase font size for whole DBeaver by changing `Main font` size in preferences.

**Comment by GregGruber** [comment]
I am using version Version 22.3.5.202302180832 on a Mac, and CMD+ does not increase the font size in the SQL editor. Is this a bug, or is there another way to increase font size?

I have gone through all of the font size settings in the User Interface | Appearance, and none of them affect the SQL Editor.

**Reply by McKAnt** [reply]
I found that the setting here:
Preferences -> User Interface -> Appearance -> Colors and Fonts -> DBeaver Fonts -> Monospace font
is the one, but it doesn't always work. Sometimes I have to set it, close DBeaver, and reopen to see the change.

Ditto for the keyboard zoom shortcuts.

**Reply by E1izabeth** [reply]
We're trying to improve font changing UX. If you encounter some bug on the latest DBeaver version **23.0.0**, you could help us to made DBeaver better by creating an issue.

**Reply by GregGruber** [reply]
Hi @E1izabeth I'm using v23.0. I find i can change the font in the settings, but it only takes effect if I exit dBeaver and launch again (what @McKAnt found).

**Reply by E1izabeth** [reply]
Issue about this bug #19354
We will try fixing it in the near releases.

**Comment by WallaceRepo** [comment]
Dbeaver UI Font is too small, giving pressure on seeing what is what. I am moving to the alternate apps. Preference does  not have an option for font size change.

**Reply by E1izabeth** [reply]
DBeaver do have an option to change font size
![image](https://user-images.githubusercontent.com/28875055/224709335-7d8bf138-2e85-41b2-8fe4-ded9689a23ec.png)

**Reply by franzolin** [reply]
Thank's

**Comment by wjatscheslaw1986** [comment]
The issue has been opened https://github.com/dbeaver/dbeaver/issues/19412

**Comment by arif98741** [comment]
I was facing same issue in my win 10. After a lot of troubles I reset my UI settings from help menu and increase font using **Ctrl+** and **Ctrl-**

![image](https://user-images.githubusercontent.com/17213478/228064088-a8de307b-ef45-439a-a23c-80463fadf960.png)

**Comment by jeff-bowles** [comment]
I don’t think the SQL editor window uses the value in Monospace font.  Try changing it to something obviously different, like 16 point size.   The SQL window on my computer remains unchanged.
